doc/manual/manual.tex

```diff
@@ -320,12 +320,26 @@ \subsection{Referencing \aspect{}}
 }
 \end{lstlisting}
 
-
+Cite the  most recent release of ASPECT (v1.5.0) as:
+\begin{lstlisting}[frame=single,language=tex]
+@misc{aspect-doi-v1.5.0,
+  author       = {Wolfgang Bangerth and
+                  Juliane Dannberg and
+                  Rene Gassmoeller and
+                  Timo Heister and
+                  others},
+  title        = {{ASPECT} v1.5.0},
+  month        = {mar},
+  year         = {2017},
+  doi          = {10.5281/zenodo.344623},
+  note         = {doi:10.5281/zenodo.344623},
+  url          = {https://doi.org/10.5281/zenodo.344623}
+}\end{lstlisting}
 
 Updated citation information can also be found on the \aspect{}
 website at \url{https://aspect.dealii.org/}.
 
-
+Citation information for recent and past versions of ASPECT can also be found at: \url{geodynamics.org/abc}.
```

Comment:
this does not show you directly how to cite ASPECT. How about we link to http://aspect.dealii.org/publications.html here and modify http://aspect.dealii.org/publications.html to also link to ABC? Maybe we should discuss first.